airbyte-integrations/connectors/source-infinicept/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,13 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "transactions",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"]
```

Comment:
I totally missed the DateFilter param while looking at the API. However, I think there maybe a case for doing incremental later since we only have one tenant with only 16 records right now for `transactions.` That said maybe Jeremy Lee can chime on this if he foresees more data to be there where incremental will be useful.

Reply:
Makes sense -- I can also explore incremental while doing the next endpoint.